tests/ui_tests/test_cluster_list_page.py

```diff
@@ -209,6 +210,52 @@ def check_components_host_info(host_info: ComponentsHostRowInfo, name: str, comp
     check_host_value('components', host_info.components, components)
 
 
+# !===== Funcs =====!
+
+
+def cluster_with_service(sdk_client: ADCMClient, path) -> Cluster:
+    """
+    Prepared cluster for test: create cluster, couple services and couple components.
+    """
+    bundle = sdk_client.upload_from_fs(path)
+    cluster = bundle.cluster_create(name=f"Test cluster {random_string()}")
+    return cluster
+
+
+@allure.step("Prepare cluster and get config")
+def prepare_cluster_and_config(sdk_client: ADCMClient, path, app):
+    """Upload bundle, create cluster and get config"""
+    cluster = cluster_with_service(sdk_client, path)
+    config = ClusterConfigPage(app.driver, app.adcm.url, cluster.cluster_id).open()
+    config.wait_page_is_opened()
+    return cluster, config
```

Comment:
"get config" is ambiguous here, because you open and return config page, so maybe clarify it a bit?
Like "Prepare cluster and open cluster config page" in step and clarify in docstring (or via typehints) that the config page is opened and returned

Reply:
renamed